TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I will remove as much as I can remove here. making it as minimal as passible while still building, and when @davkean 's template comes in we should probably switch to that


I think there's probably things being lost here from a tooling experience perspective.